### PR TITLE
Task 1-Pull request

### PR DESCRIPTION
**Pull request by AhmedBamatraf**
We added the EDA, and finished the task 1:
- We applied geohash to use a categorical representation of the location rather than using the Lat/Long, which helps us in the stratified sampling. (Done by Aseel)
- Normalized the features (Done by Aseel)
- Tested different DBSCAN parameters to get good clusters -silhouette score > 0.5. (Done by Ahmed)
- Plotted the clusters of each tested parameters of DBSCAN (Done by Ahmed)

### CODE REVIEW

File: Project_File.ipynb
Comment (IsamAljawarneh) [inline Project_File.ipynb:1]: what do you mean by `stratified_sample, _ = train_test_split(data, test_size=1 - sample_size, stratify=data['geohash'], random_state=42)`. This is not the correct way to take a `stratified sample`. Stratified sampling by `geohash` as strata means that we first group by `geohash` value, then we take a percentage of tuples from each geohash bracket independently.
Anyway, this is your work and I am merging those updates into the main as per your request.
For the clustering part, in your report try to discuss how changing the configurable parameters `epsilon` and `samples` have a discernible effect on the `Silhouette Score` score and the overall clustering performance, you have already varied those parameters so try to discuss the results in your report.

**Review by IsamAljawarneh — APPROVED** [review]
what do you mean by `stratified_sample, _ = train_test_split(data, test_size=1 - sample_size, stratify=data['geohash'], random_state=42)`. This is not the correct way to take a `stratified sample`. Stratified sampling by `geohash` as strata means that we first group by `geohash` value, then we take a percentage of tuples from each geohash bracket independently.
`Anyways`, this is your work and I am merging those updates into the main as per your request.
For the clustering part, in your report try to discuss how changing the configurable parameters `epsilon` and `samples` have a discernible effect on the `Silhouette Score` score and the overall clustering performance, you have already varied those parameters so try to discuss the results in your report.
